airgun/views/syncplan.py

```diff
@@ -73,7 +74,9 @@ class details(SatTab):
         next_sync = ReadOnlyEntry(name='Next Sync')
         enabled = EditableEntryCheckbox(name='Sync Enabled')
         interval = EditableEntrySelect(name='Interval')
+        cron_expression = EditableEntry(name='Cron Logic')
```

Comment:
just change order to have fields in the same way we have them on UI:
```
name = EditableEntry(name='Name')
description = EditableEntry(name='Description')
date_time = EditableDateTime(name='Start Date')
next_sync = ReadOnlyEntry(name='Next Sync')
recurring_logic = ReadOnlyEntry(name='Recurring Logic')
enabled = EditableEntryCheckbox(name='Sync Enabled')
interval = EditableEntrySelect(name='Interval')
cron_expression = EditableEntry(name='Cron Logic')
products_count = ReadOnlyEntry(name='Products')
```

Reply:
Done!